<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptchaDialog</class>
 <widget class="QDialog" name="CaptchaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plase Solve the Captcha</string>
  </property>
  <widget class="QLineEdit" name="captchaEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please type the image text here</string>
   </property>
  </widget>
  <widget class="QLabel" name="captchaLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>   Captcha image will be place here, please wait a moment !</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Captcha :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>230</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshCaptchaButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
